src/styles/themes/dark.css

```diff
@@ -72,7 +72,7 @@
 	--token-color-foreground-action: #2b89ff;
 	--token-color-foreground-action-hover: #389aff;
 	--token-color-foreground-action-active: #4ca1ff;
-	--token-color-foreground-highlight: #aa3fff;
+	--token-color-foreground-highlight: #ab42ff;
```

Comment:
This will reference `var(--token-color-palette-200)` after https://github.com/hashicorp/dev-portal/pull/1744